### ISSUE TITLE
1-4 does not update the priority

### ISSUE THREAD

**Issue by dgrtbh** [mgsloan/todoist-shortcuts#273]
When I'm inside a task list, I expect that by clicking 1-4, the task's priority will be updated. Currently, nothing is happening.

**Comment by mgsloan** [comment]
Curious, it works for me in agenda view, label view, filter view, and project view.  What d you mean by "inside a task list"?

**Comment by dgrtbh** [comment]
It's weird. I just checked and it seems that I can switch to another prio when task is prio 1 (red), but cannot do it the other way (switch back to red prio). Can you confirm you can switch between any status to any other status on your end?

**Comment by mgsloan** [comment]
For me all priority switching works, doesn't get stuck

**Comment by dgrtbh** [comment]
I decided to try one more thing, so I enabled "experimental features" in my web app. It worked as expected, but as soon as I turned the experimental features off, it broke once again. Do you think the current state of things in the beta/experimental stream will eventually make it to production?

**Comment by mgsloan** [comment]
Ahh, interesting!  Yes quite probably so, I imagine it is a bug fix in the
beta stream

**Comment by mlhdeveloper** [comment]
@dgrtbh You probably already tried this, but, just in case, have you tried disabling then re-enabling the extension? I was having issues with some functionality after a recent Todoist update and that just fixed them for me.

**Comment by error9900** [comment]
This is an old Issue, but since it's still `Open` and related I figured I'd add this here instead of creating a new Issue:

I think a recent Todoist update broke using numbers to set the Priority...

In the List view, `1`, `2`, `3`, & `4` curiously enter Edit mode for the task.

In the Task view, `1`, `2`, `3`, & `4` seem to select the Priority "field", but it doesn't change the value. However, if I hit the number key a second time, it does set the Priority value appropriately...

I tried disabling then re-enabling the Extension, then refreshing Todoist, but that didn't fix it.

Todoist Version: v8366
Extension Version: 202

**Comment by MATV0** [comment]
> This is an old Issue, but since it's still `Open` and related I figured I'd add this here instead of creating a new Issue:
> 
> I think a recent Todoist update broke using numbers to set the Priority...
> 
> In the List view, `1`, `2`, `3`, & `4` curiously enter Edit mode for the task.
> 
> In the Task view, `1`, `2`, `3`, & `4` seem to select the Priority "field", but it doesn't change the value. However, if I hit the number key a second time, it does set the Priority value appropriately...
> 
> I tried disabling then re-enabling the Extension, then refreshing Todoist, but that didn't fix it.
> 
> Todoist Version: v8366 Extension Version: 202

Yes I've been having this problem for a week or two now. Also V doesn't offer to move a task, instead it selects it, but if I press V a second time then the move prompt comes up. Frustrating but at least I can work around it for now, but please fix!

**Comment by mgsloan** [comment]
Fixed in version 203
